scene/resources/particles_material.h

```diff
@@ -282,6 +299,9 @@ class ParticlesMaterial : public Material {
 
 	void set_gravity(const Vector3 &p_gravity);
 	Vector3 get_gravity() const;
+	
```

Comment:
Another invisible tab.

These three are the only things left that Travis CI is complaining about.